_parcel/behaviors/search-form.js

```diff
@@ -9,7 +9,12 @@ onmount('[data-js-search-form]', function () {
   on(this, 'submit', (e) => {
     e.preventDefault()
 
-    const link = document.querySelector('a[data-search-index]:visible')
+    const links = document.querySelectorAll('a[data-search-index]')
+
+    const link = [].filter.call(links, function (el) {
+      var style = window.getComputedStyle(el);
+      return (style.display !== 'none')
```

Comment:
Thanks for this! I do wonder if using a class name or data attribute would be better here - using getComputedStyle is expensive.

Reply:
Yes, no problem. Now the search function uses `hidden` attribute instead of `aria-hidden`, and the search form uses that attribute instead of CSS properties. Let me know if you see anything else you would like me to change.